kornia/geometry/boxes.py

```diff
@@ -222,6 +299,8 @@ def to_tensor(self, mode: str = "xyxy") -> torch.Tensor:
                   box width and height are defined as ``width = xmax - xmin`` and ``height = ymax - ymin``.
                 * 'vertices_plus': similar to 'vertices' mode but where box width and length are defined as
                   ``width = xmax - xmin + 1`` and ``height = ymax - ymin + 1``. ymin + 1``.
+            as_padded: whether to keep the pads for a list of boxes. This parameter is only valid
```

Comment:
```suggestion
            padded: whether to keep the pads for a list of boxes. This parameter is only valid
```

Reply:
I will go `as_padded_sequence`. `padded` is confusing here.